conf/create-filtered-config.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @fileoverview Helper to create a config based on a filter from the core rules
+ * @author Teddy Katz
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const load = require("../lib/load-rules"),
+    rules = require("../lib/rules");
+
+//------------------------------------------------------------------------------
+// Public Interface
+//------------------------------------------------------------------------------
+
+
+/**
+ * Creates a config from a filtered list of core rules
+ * @param {Function} ruleFilter A filter function for rules. This gets passed the rule object as the first argument, and
+ * the rule id as the second argument. Returns a truthy value if the rule should be included, and a falsy value otherwise
+ * @returns {Object} A frozen config with the given core rules enabled
+ */
+module.exports = function(ruleFilter) {
+    return Object.freeze({
```

Comment:
Same as the other PR. `Object.freeze` is really slow and impacts performance in a major way.